pipeline.yml

```diff
@@ -880,10 +879,9 @@ jobs:
       resource: masterbosh-state
   - task: decrypt-private-key
     file: pipeline-tasks/decrypt.yml
-    config:
-      inputs:
-      - name: pipeline-tasks
-      - name: common-masterbosh
+    input_mapping:
+    - name: pipeline-tasks
```

Comment:
🤔 seems so. Does this need to be updated to [match something like this](https://github.com/18F/cg-deploy-bosh/blob/master/update-runtime-config.yml#L8-L20)? 

```yaml
input_mapping:
- {name: pipeline-tasks}
- {name: common-masterbosh}
```